### DISCUSSION TITLE
Login page move `forgot password`

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2257] [Ideas / Feature Requests]
In the sign in page, there are two text fields, `mail` and `password`, but there is a button `Forgot Password?` in between.

When navigating the site with the keyboard, first the `mail` is typed in, then `tab` pressed, which highlights `Forgot Password?`, then `tab` again, to finally get to the `password`. This means users have to double click `tab` in order to reach the next textbox. 

This could be improved by moving the `Forgot Password?` hyperlink to below the `password` box. 

![screenshot_20210205_104830](https://user-images.githubusercontent.com/30683012/107017695-bc354200-679f-11eb-8726-bec72b118953.png)

**Comment by pavlenex** [comment]
Good idea!

**Reply by dennisreimann** [reply]
Done in #2258.

**Reply by HeadyWook** [reply]
I do this all the time. Good suggestion.
